apps/docs-website/docs/vscode-support.md

```diff
@@ -38,10 +38,18 @@ For [Astro](https://astro.build/) projects, add the astro extension too:
 
 ## Avoid "source.organizeImports"
 
-Sheriff already handle imports sorting, so you have to disable thew VSCode automatic import sorting feature to avoid conflicts:
+Sheriff already handle imports sorting, so you have to disable VSCode automatic import sorting feature to avoid conflicts:
 
 ```JSONC title=".vscode/settings.json"
 "editor.codeActionsOnSave": {
     "source.organizeImports": false
```

Comment:
actually false will be deprecated. We shall recommend `never`

Reply:
Oh, yeah, I should've caught that.